setup.py

```diff
@@ -98,7 +98,7 @@ def is_requirement(line):
             'modernize_node_workflow = edx_repo_tools.codemods.node16.gha_ci_modernizer:main',
             'modernize_openedx_yaml = edx_repo_tools.modernize_openedx_yaml:main',
             'modernize_setup_file = edx_repo_tools.codemods.django3.setup_file_modernizer:main',
-            'modernize_tox = edx_repo_tools.codemods.django3.tox_modernizer:main',
+            'modernize_tox = edx_repo_tools.codemods.python312.tox_modernizer:main',
```

Comment:
instead of replacing, why not create a new `modernize_tox_py312`?

Reply:
Updated.